src/style.css

```diff
@@ -0,0 +1,129 @@
+body,
+h1,
+h2,
+h3,
+button {
+  margin: 0;
+  padding: 0;
+  font-size: inherit;
+}
```

Comment:
body can inherit only from html and don't have default paddings. try to apply styles only where needed